tests/amountClaim/enterAmountClaimDetails_test.js

```diff
@@ -31,6 +32,40 @@ Scenario('I can fill in Claimant, Defendant and Claim amount details', (I, userS
   amountClaimSteps.verifySubmittedPage()
 })
 
+Scenario('I can fill in multiple Claimants, Defendant and Claim amount details', (I, userSteps, defendantSteps, amountClaimSteps) => {
+  userSteps.loginDefaultUser()
+  userSteps.startClaim()
+  userSteps.enterYourOrganisationNamePage()
+  userSteps.enterYourOrganisationAddress()
+  userSteps.enterYourOrganisationContactDetails()
+  userSteps.enterYourReferenceNumber()
+  userSteps.enterYourPreferredCountyCourt()
+  userSteps.enterClaimantTypeOrganisation()
+  I.see('Claimant: Abc corporation')
+  userSteps.enterClaimantAddress()
+  userSteps.addAdditionalClaimant()
+  userSteps.enterClaimantTypeIndividual()
+  I.see('Claimant 2: Mr Benugo')
+  userSteps.enterClaimantAddress()
+  userSteps.noAdditionalClaimant()
+  defendantSteps.enterDefendantTypeOrganisation()
```

Comment:
As long as you add multiple clement and verify it on add more clements page should be fine
otherwise it will increase time on pipeline.